### PR TITLE
Enhancements: Updated README, certain changes done.

### PR DESCRIPTION
**Pull request by iemafzalhassan**
## Motivation and Context

The motivation behind these changes was to enhance the visibility and engagement of the repository by updating the README.md file, adding social media account badges, and integrating a repository rater. These changes aim to make the repository more accessible to users and encourage interaction with its content.

## Description

- Updated the README.md file to provide a comprehensive overview of the repository, including its purpose, features, and usage instructions.
- Added badges linking to social media accounts (Facebook and Twitter) to facilitate direct access to the maintainer's profiles.
- Integrated a repository rater to gather feedback and ratings from users, enhancing the repository's credibility and trustworthiness.

## How Has This Been Tested?

These changes have been tested by:

- Reviewing the updated README.md file to ensure clarity and completeness of information.
- Clicking on the social media badges to confirm that they correctly link to the respective profiles.
- Interacting with the repository rater to validate its functionality and usability.

## Screenshots (if appropriate):

Screenshot 2024-05-17 at 3 41 29 AM

## Types of Changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: assets/images/hero.png
Comment (adityaraute) [inline assets/images/hero.png:1]: Where is this screenshot from? The text on the right could be improved.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
+## Get Involved
+
+We warmly invite individuals who share our vision and passion for community empowerment to join us in our mission. There are several meaningful ways you can contribute and become actively involved in our efforts:
+
+## Contact us
+
+  - **Email**: RBJKS.info@gmail.com
+  - **GitHub:** [RBJKS GitHub](https://github.com/rbjks)
+  - **Address:** PO: Jassur, Tehsil - Nurpur, District - Kangra, HP, 176201.
+  - **Call us:** +91 - 9418151074 / 8675300009
```
Comment (adityaraute) [inline README.md:42]: We don't need to mention the Phone numbers on the README file.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,42 @@
-# rbjks.github.io
-<a href="https://gitpod.io/#https://github.com/rbjks/rbjks.github.io" target="_blank">
-  <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open RBJKS in Gitpod">
-</a>
+[![RBJKS NGO](assets/images/hero.png)](https://rbjks.github.io/index.html)
 
+[![RepoRater Badge](https://repo-rater.eddiehub.io/api/badge?owner=rbjks&name=rbjks.github.io)](https://repo-rater.eddiehub.io/rate?owner=rbjks&name=rbjks.github.io)
```
Comment (adityaraute) [inline README.md:3]: Please remove this badge as it is part of another issue.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,42 @@
...
+  - About
+  - Mission
+  - Get Involved
+  - Contact
 
-<span style="color:#edc9af">Desert Sand - #edc9af</span>
+## About
 
-![Dessert Sand](https://user-images.githubusercontent.com/115361239/220578344-261f5716-3908-427b-9b17-a5e3215a18c6.png)
+RBJKS, a distinguished non-profit organization headquartered in Nagpur, Himachal Pradesh, India, stands as a beacon of hope and progress for our local community. With a steadfast commitment to social responsibility and sustainable development, we endeavor to address pressing issues that affect the lives of individuals and families.
```
Comment (adityaraute) [inline README.md:23]: Use the full name instead of 'RBJKS'.
Also, it's Nurpur not Nagpur.

File: README.md [hunk 4]
```diff
@@ -1,24 +1,42 @@
...
-<span style="color:#22B9A8">Emerald - #22B9A8</span>
+# [RBJKS NGO](https://rbjks.github.io/index.html)
 
-![Emerald](https://user-images.githubusercontent.com/115361239/220578047-1ed3c681-0282-4b1e-85bd-66e86b1344bd.png)
+Welcome to the RBJKS (Ranjeet Bakhshi Jan Kalyan Sabha) NGO Project! Our mission is to address local community issues in Nagpur, Himachal Pradesh, India, through organic crowdfunding initiatives.
 
-<span style="color:#ff9933">Deep Saffron - #ff9933</span>
+## Table of Contents
 
-![Deep Saffron](https://user-images.githubusercontent.com/115361239/220579735-fb87b37d-67f4-406c-8188-2d42f29f9847.png)
```
Comment (adityaraute) [inline README.md:20]: Please don't remove the color schemes as these are important to contributors. 
Perhaps we can add them as another section just below the Contact section?

**Review by adityaraute — CHANGES_REQUESTED** [review]
I've made some suggestions.
Remember, the README is more for the repository and less for the NGO's functions. It should serve as a guide to our contributors first.

**Review by adityaraute — APPROVED** [review]
This looks really good!

**Comment by adityaraute** [comment]
Did you use any AI tool to generate the content?

**Comment by iemafzalhassan** [comment]
Yup! took support from [chatGPT](https://chatgpt.com/?model=gpt-4o) and [blackBOX](https://www.blackbox.ai/)

**Comment by adityaraute** [comment]
Cool, thanks for sharing.

**Comment by iemafzalhassan** [comment]
**You're welcome** 🤗

**Pull request merged**